internal/k8s/ako_init.go

```diff
@@ -707,11 +707,14 @@ func (c *AviController) addIndexers() {
 		informer.AviInfraSettingInformer.Informer().AddIndexers(
 			cache.Indexers{
 				lib.SeGroupAviSettingIndex: func(obj interface{}) ([]string, error) {
-					infraSetting, ok := obj.(*akov1alpha1.AviInfraSetting)
+					seg := ""
+					infraSetting, ok := obj.(*akov1beta1.AviInfraSetting)
 					if !ok {
 						return []string{}, nil
+					} else {
+						seg = infraSetting.Spec.SeGroup.Name
```

Comment:
Instead of this `else` statement we can return []string{infraSetting.Spec.SeGroup.Name} (same as previous).